google-http-client-gson/src/main/java/com/google/api/client/json/gson/GsonFactory.java

```diff
@@ -90,4 +92,22 @@ public JsonGenerator createJsonGenerator(OutputStream out, Charset enc) {
   public JsonGenerator createJsonGenerator(Writer writer) {
     return new GsonGenerator(this, new JsonWriter(writer));
   }
+
+  /** Returns true if it is lenient to input JSON value. */
+  boolean getReadLeniency() {
+    return readLeniency;
+  }
+
+  private static GsonFactory newInstance(GsonFactory gsonFactory) {
```

Comment:
As per Effective Java's clone section, creating a factory method. I don't think a subclass needs to access this method.